jkube-kit/jkube-kit-helidon/pom.xml

```diff
@@ -65,6 +65,11 @@
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this dependency added to this module?

Reply:
removed